src/new-components/tooltip/tooltip.module.css

```
@@ -1,4 +1,8 @@
+:root {
+    --reactist-tooltip-zindex: 1000;
+}
+
 .tooltip {
     text-overflow: ellipsis;
-    z-index: 1000;
+    z-index: var(--reactist-tooltip-zindex);
```

_(optional but encouraged for consistency)_

We already have a convention of not declaring its value, and use a default value when the variable is used. See my next comment below.

```suggestion
    z-index: var(--reactist-stacking-order-tooltip, 1000);
```

This includes removing the declaration in `:root`.

Alternatively, we might decide to move in the other direction and declare all stacking order variables explicitly. No need to do it in this PR, though.

---

I went with your second suggestion and pulled all the stacking-order properties over to `default-styles.css` 👍